packages/component-utils/src/styleVariables.js

```diff
@@ -15,6 +15,11 @@
  */
 
 /* @flow */
+export const blackAndWhite = {
```

Comment:
Following how we handled the rest of the palette.   It seems like `black` and `white` should just be keys inside the `color` object in `styleVariables`.  Then, in `mineralTheme`, we should define `color_black` and `color_white` using those keys.